python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -8,6 +8,7 @@
 except ImportError:
     import unittest
 
+ganga_test_dir = 'gangadir testing'
```

Comment:
I would call this `ganga_test_dir_name` so make it clear that it's not the full directory but rather only a fragment.